tests/test_potential.py

```diff
@@ -3285,33 +3285,16 @@ def __init__(self,rc=0.75,maxt=1.,nt=50):
         times= numpy.linspace(0.,maxt,nt)
         o1.integrate(times,lp,method='dopr54_c')
         o2.integrate(times,lp,method='dopr54_c')
-        self._o1p= potential.MovingObjectPotential(o1)
-        self._o2p= potential.MovingObjectPotential(o2)
+        oplum = potential.PlummerPotential(amp=0.06, b=0.01)
+        self._o1p= potential.MovingObjectPotential(o1, pot=oplum)
+        self._o2p= potential.MovingObjectPotential(o2, pot=oplum)
         testMWPotential.__init__(self,[self._o1p,self._o2p])
         self.isNonAxi= True
         return None
     def phi2deriv(self,R,z,phi=0.,t=0.):
         raise AttributeError
     def OmegaP(self):
         return 1./self._rc
-from galpy.potential.ForceSoftening import PlummerSoftening
```

Comment:
I don't think this test should be removed, because it should now just explicitly set ``pot=Plummer...`` to mimic what what was going on before.